package.json

```diff
@@ -239,5 +243,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "lint-staged": {
+    "*": "yarn eslint --ext .js,.jsx,.ts,.tsx --fix"
```

Comment:
`lint-staged` gets passed a list of all changed files like `<cmd> <file>[, <file>, ...]`, and I've found in the past it's a little easier to let respective CLIs handle exclusions or extensions than configure here in JSON — lmk if y'all disagree